pom.xml

```diff
@@ -99,6 +107,18 @@
 	</build>
 	<profiles>
 		<profile>
+			<id>weld.test.3plus</id>
+			<activation>
+				<property>
+					<name>weld.test.3plus</name>
```

Comment:
Ehh, probably, but then it's another thing to maintain when new versions of Weld 3 come out.

It would probably be best to default to 3.x for tests anyway, which means the weld.test.3plus profile would have to be on by default. But Maven is kind of a pain with profiles, plus I couldn't think of a good name for the inverse property. But I've gone ahead and it seems to work now. 

What do you think of `-Dweld.test.1or2` ?